go/useeioapi/models.go

```diff
@@ -32,7 +33,8 @@ func ReadModelInfos(dataDir string) ([]*ModelInfo, error) {
 			Name:         row[1],
 			Location:     row[2],
 			Description:  row[3],
-			SectorSchema: row[4]})
+			SectorSchema: row[4],
+			Hash: row[5]})
```

Comment:
the check above needs to be changed to `len(row) < 6` for this additional column then; we may should also return an error when the format is not correct instead of  skipping these rows?

Reply:
Addressed in this commit.  I agree to print an error message and return nil if there is an error in the format
https://github.com/USEPA/USEEIO_API/commit/ec0a7ca11f6bee68cb9882d46b2ee6bf29678648